src/components/chip-set/chip-set.tsx

```diff
@@ -177,7 +177,7 @@ export class ChipSet {
      * Dispatched when the input is changed for type `input`
      */
     @Event()
-    private input: EventEmitter<string>;
+    private chipSetInput: EventEmitter<string>;
```

Comment:
Two questions:

1. Should we name it `chipSetInput` or `limelChipSetInput`?
2. This is the only one that we have to update because it causes build errors with Stencil v3, but should we rename other events too while we're at it?
  - Argument for: We would like to change the names, at least for all events that are shadowing native events, and perhaps for all events, to avoid future conflicts. It's always a horrible struggle to get people to update their customisations, let's do all the changes we know we want at once.
  - Argument against: It's always a horrible struggle to get people to update their customisations, so let's not make this any more difficult than it has to be. If we force too many changes at once, people are probably just going to protest and refuse instead.